client/state/rewind/retention/test/reducer.js

```diff
@@ -0,0 +1,40 @@
+import {
+	JETPACK_BACKUP_RETENTION_UPDATE,
+	JETPACK_BACKUP_RETENTION_UPDATE_ERROR,
+	JETPACK_BACKUP_RETENTION_UPDATE_SUCCESS,
+} from 'calypso/state/action-types';
+import retentionReducer from '../reducer';
+
+describe( 'rewind.retention reducers', () => {
+	describe( 'updateBackupRetentionRequestStatus', () => {
+		it.each( [
+			{
+				state: undefined,
+				action: {},
```

Comment:
Is that a realistic test? Will we ever have an action without a type? I'd suggest adding a random type here to make it a bit more realistic.